pom.xml

```diff
@@ -227,8 +227,8 @@
         <updatePolicy>daily</updatePolicy>
         <checksumPolicy>fail</checksumPolicy>
       </snapshots>
-      <id>sonatype-nexus-snapshots</id>
-      <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+      <id>apache-snapshots</id>
+      <url>https://repository.apache.org/content/groups/snapshots</url>
```

Comment:
I removed the repositories section in 33ddcb77bd50de.  It was created for sonatype and should be revisited if we ever depend on a SNAPSHOT